src/composables/map/ol.composable.ts

```diff
@@ -255,6 +256,7 @@ export default function useOpenLayers() {
         olMap.addLayer(bgBaseLayer)
       }
     }
+    statePersistorStyleService.restoreStyle()
```

Comment:
Is this necessary, additionally to calling `bootstrapStyle()`?

Reply:
yes, because it will restore the style from local storage which can be different for various background layers

Reply:
ok, didn't really understand why the `restoreStyle()` called within `bootstrapStyle()` is not sufficient